TicketsDemo.Mongo/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <connectionStrings>
+    <add name="MongoDb" connectionString="mongodb://localhost:27017/TicketsDemo" />
```

Comment:
The startup project is TicketsDemo (not the TicketsDemo.Mongo). 

Therefore, in the runtime settings are being read from TicketsDemo and subsequently this setting will be never read.